### ISSUE TITLE
uriparser dependency build fail

### ISSUE THREAD

**Issue by th0ma7** [SynoCommunity/spksrc#4681]
(for tracking purpose)

The download url for `uriparser` changed.
Also the version 0.9.1 is declared "buggy".

Many options, starting with fixing the download URL to:
`PKG_DIST_SITE = https://sourceforge.net/projects/$(PKG_NAME)/files/Sources/$(PKG_VERS)`

- Easy to downgrade to version 0.90 + small PLIST adjustment.
- Otherwise requires migrating to `cmake` for newer version and fails miserable on googletest
- still use the .zip (instead of .tar.gz) file of version 0.91

**Comment by th0ma7** [comment]
Note that I tried to simply install googletest (and libgtest-dev) in the image but it didn't worked.
I may try using `native/` or better, disabling it using `cmake` params (if any)

Also, reading on https://github.com/google/googletest it says the following:
`GoogleTest requires a codebase and compiler compliant with the C++11 standard or newer.`

This means that if we ever need to support per-arch googletest = failure to build on some DSM6 arch due to old gcc version.

**Comment by th0ma7** [comment]
Fix ready:  #4682
```
CMAKE_ARGS  = -DCMAKE_BUILD_TYPE=Release
CMAKE_ARGS += -DURIPARSER_BUILD_DOCS=OFF
# Requires Gtest
CMAKE_ARGS += -DURIPARSER_BUILD_TESTS=OFF

include ../../mk/spksrc.cross-cmake.mk
```
